### ISSUE TITLE
Prisma Export: Invalid JSON

### ISSUE THREAD

**Issue by luhagel** [prisma/prisma#3168]
**Describe the bug**
As of right now, the `prisma export` command produces invalid data when it comes to json(list) fields resulting in an error when trying to import that data somewhere else.

**To Reproduce**
1. Spin up a prisma stage with some form of json list inside
2. Export the data with `prisma export`
3. Try to import that data into another stage with `prisma import`

```
prisma import --data export-2018-09-21T09:00:40.144Z.zip
Unzipping 31ms
Uncaught exception, cleaning up: Error: Value "{\"prop\":\"whatever\",\"label\":\"Label\"}" for field personalDownloads is not a valid Json
Validating data ✖
```

**Expected behavior**
The import should work without any hiccups

**Versions (please complete the following information):**
 - OS: Tested on `OS X Mojave`& `Linux 4.18.8` 
 - `prisma` CLI: `prisma/1.16.4 x64 node-v10.11.0`
 - Prisma Server: `1.16.0`

**Additional context**
Extracting the zip, removing the list contents from the lists folder and compressing everything again fixes the problem.

**Comment by luhagel** [comment]
Quick update, just did some further testing, it looks like the problem lies within the fact that the export creates escaped json strings instead of objects. Removing all the backslashes and superflous quotation marks from the list before zipping everything up again works as expected, with the data being where it belongs.

Current:
```
{
  "valueType": "lists",
  "values": [
    {
      "_typeName": "Semester",
      "id": "myid",
      "personalDownloads": [
        "{\"prop\":\"test_1\",\"label\":\"Test 1\"}",
        "{\"prop\":\"test_2\",\"label\":\"Test 2\"}"
      ]
    }
  ]
}
```
Working:
```
{
  "valueType": "lists",
  "values": [
    {
      "_typeName": "TName",
      "id": "myid",
      "field": [
        {
          "prop": "test_1",
          "label": "Test 1"
        },
        {
          "prop": "test_2",
          "label": "Test 2"
        }
      ]
    }
  ]
}
```

**Comment by marktani** [comment]
Thanks for the follow up. It is my current understanding that this is a mismatch between the output format for JSON values by the backend, and the JSON validation done by the CLI.

Both components should agree on the exact parsing behaviour. I am not sure which is per the spec, and which is not.

**Comment by divyenduz** [comment]
The quotes are generated from backend for [legacy reasons](https://github.com/prisma/prisma/blob/72c97c47b748457b3c59627e66090d94e0c2a8f9/server/servers/api/src/main/scala/com/prisma/api/import_export/BulkExport.scala#L105-L114). 

Bringing import/export in sync would be a breaking change but it is important and [NDF compliant](https://www.prisma.io/docs/1.17/prisma-cli-and-configuration/data-import-and-export-jsw9/#normalized-data-format) as NDF does not mention escape quoting for anything.

**Comment by do4gr** [comment]
Thanks for reporting this @luhagel , we just released 1.17.2 which fixes the issue by not stringifying the output anymore.
